### ISSUE TITLE
The program didn't find eggnog table

### ISSUE THREAD

**Issue by LujingF** [jhcepas/eggnog-mapper#77]
Hi, When I use the test example command: python emapper.py -i test/p53.fa --output p53_maNOG  --cpu 4 -m diamond . The diamond was ok while the program was ended with an error 'sqlite3.OperationalError: no such table: eggnog'. But I had already downloaded eggnog.db in data folder, so how to solve this problem? Should I set this in environment variable path or edit database configure file(I don't known if this kind of file exists in your program) or some other ways. A little suggestion: a more specific manual would be better and easy to use for beginners.

**Comment by jhcepas** [comment]
are you on a shared filesystem? sqlite does not work well under NFS

**Comment by LujingF** [comment]
well, I use the program on my PC not a shared filesystem. It's ok to run 'sqlite3 eggnog.db'

**Comment by jhcepas** [comment]
can you copy here some info about your system? python version, SO version, eggnog-mapper version etc.

**Comment by LujingF** [comment]
ubuntu 16.04 
Python 2.7.14 :: Anaconda, Inc.
kernel version: Linux taishan 4.8.12-040812-generic #201612020431 SMP Fri Dec 2 09:33:31 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
eggnog-mapper version: 1.0.3
so version: /lib/x86_64-linux-gnu/ld-2.23.so

**Comment by LujingF** [comment]
Hello, jhcepas. I just figured out that problem. I reason why it failed is database which I used is not the latest one. The older version database was downloaded at http://eggnogdb.embl.de/download/latest/eggnog-mapper-data/ . The program cannot recognize the eggnog.db . Yesterday, I downloaded another database from 'http://eggnogdb.embl.de/download/emapperdb-4.5.1/' and it worked. very appreciated for helping me.
